src/ruby/Shop.java

```diff
@@ -0,0 +1,63 @@
+
+/*Create a system (Shop) which maintains each item's quantity.
+
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala�s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. 
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.
+*/
+package ruby;
+public class Shop {
+	int maggie = 50;
+	int dosa= 43;
+	int oilPouches = 39;
+	int panipuri = 43;
+	int masala = 73;
+	
+	void outOfStock(){
+		if(maggie <= 0)
```

Comment:
here, condition should be == not <=0 otherwise quantity never be in minus logically right.
improve in next PR